app/assets/stylesheets/config/_mediaqueries.scss

```diff
@@ -16,6 +16,12 @@ $width-bp-xlarge: 90em;             // = 1440px
 
 /* ====== QUERIES ====== */
 
+@mixin small-down {
```

Comment:
Could you add the 'from-' before 'small-down' to match the the rest of the mixins ?

Reply:
it's because is a breakpoint to catch every screen at 600px to down